$OpenBSD: patch-etc_icinga2_features-available_api_conf,v 1.1 2017/11/20 11:11:20 sthen Exp $

XXX Temporarily readded to assist migration (pkg_add -u will replace an
untouched old config file with a new one). Remove after OpenBSD 6.3.

Index: etc/icinga2/features-available/api.conf
--- etc/icinga2/features-available/api.conf.orig
+++ etc/icinga2/features-available/api.conf
@@ -3,6 +3,12 @@
  */
 
 object ApiListener "api" {
+  // Legacy locations to assist upgrades. Icinga now uses /var/lib/icinga/certs,
+  // but will copy files at startup if the new location has older/no files.
+  cert_path = SysconfDir + "/icinga2/pki/" + NodeName + ".crt"
+  key_path = SysconfDir + "/icinga2/pki/" + NodeName + ".key"
+  ca_path = SysconfDir + "/icinga2/pki/ca.crt"
+
   //accept_config = false
   //accept_commands = false
 
